docs/devGuide/githubActions/markbindAction.md

```diff
@@ -0,0 +1,56 @@
+{% set title = "GitHub Actions: markbind-action" %}
+<span id="title" class="d-none">{{ title }}</span>
+
+<frontmatter>
+  title: "{{ title }}"
+  layout: devGuide.md
+  pageNav: default
+</frontmatter>
+
+# {{ title }}
+
+<div class="lead">
+
+A GitHub Action that builds and deploys a MarkBind site. It helps users to streamline their workflow by:
+* Making it easier to deploy without having to write their own actions from scratch
+* Making the workflows consistent and up to date with any changes specific to MarkBind
+
+The source code is at [MarkBind/markbind-action](https://github.com/MarkBind/markbind-action).
+</div>
+
+**Follow these steps to get started on modifying the source code**
+## Setting Up
+1. Fork the [action repository](https://github.com/MarkBind/markbind-action)
+1. Learn the existing workflow syntax/options provided in the repository's [readme.md](https://github.com/MarkBind/markbind-action#readme)
+1. Set up a repository that contains a MarkBind site to test the action as a user
```

Comment:
```suggestion
1. Set up a repository that contains a MarkBind site to test the action:
```
(may be confusing - user of the action or MarkBind - might be best to leave it as is since it is implied (developer guide))